frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/clearingDetails.jspf

```diff
@@ -167,6 +201,115 @@
                 var data = $(event.currentTarget).data();
                 fossology.openSelectClearingDialog('fossologySending', data.releaseId);
             });
+
+            $('.showSpdxContentBtn').on('click', function(event) {
+                handleShowSpdxContentClick($(event.currentTarget));
+            });
+
+            function handleShowSpdxContentClick($btn) {
+                var tableData = $('#spdxAttachments').data(),
+                    rawUrl = tableData.loadSpdxLicenseInfoUrl,
+                    url = Liferay.PortletURL.createURL(rawUrl),
+                    btnData = $btn.data(),
+                    releaseId = btnData.releaseId,
+                    attachmentId = btnData.attachmentId,
+                    $row = $('#' + attachmentId),
+                    $btnCell = $row.find('td[data-attachment-id=' + attachmentId + ']');
+
+                url.setParameter(tableData.releaseIdParameterName, releaseId);
+                url.setParameter(tableData.attachmentIdParameterName, attachmentId);
+
+                replaceContentWithSpinner($btnCell);
+
+                $.ajax({
+                    type: 'GET',
+                    dataType: 'json',
+                    url: url.toString(),
+                }).done(function(result) {
+                    if (!result || result.length == 0 || Object.getOwnPropertyNames(result) == 0) {
```

Comment:
You should add an `.length` to getOwnPropertyNames as it returns an array.

Reply:
done :-)